manifest.json

```diff
@@ -444,6 +444,39 @@
                 }
             }
         },
+        "net.eia485.InspectorDelegateCaller": {
+            "name": "InspectorDelegateCaller",
+            "description": "adds some buttons to inspector ui for workers that have callable delgates",
+            "category": "Inspectors",
+            "sourceLocation": "https://github.com/EIA485/NeosInspectorDelegateCaller",
+            "authors": {
+                "eia485": {
+                    "url": "https://github.com/EIA485",
+                    "iconUrl" : "https://avatars.githubusercontent.com/u/17285570"
```

Comment:
Was about to say, that wasn't part of the schema but I found it was recently added to it https://github.com/neos-modding-group/neos-mod-manifest/commit/73fd82a52d727a81d116533bc85ac78fd3c2d1ea